cli/azd/internal/vsrpc/testdata/manifests/aspire-starter.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I will add a script to regenerate this based on the sample in `samples/aspire-starter.json`